src/log4stash/Authentication/AwsAuthenticationMethod.cs

```diff
@@ -15,20 +18,26 @@ public class AwsAuthenticationMethod : IAuthenticationMethod
 
         public string CreateAuthenticationHeader(RequestData requestData)
```

Comment:
Can we delete `CreateAuthenticationHeader` ? I didn't find any reference to it :)